### ISSUE TITLE
[BUG] - Log buffering issue

### ISSUE THREAD

**Issue by erikd** [input-output-hk/cardano-node#3149]
**Internal**

**Area**
*Logging*

**Summary**
When the node is shutdown improperly (eg machine or VM it runs in dies) and then restarted there are some initial log messages then a seeming hang for a lttle over two hours before the stdout log messages are finally printed (with the log time stamps being from near the start of the seeming hang.

**Steps to reproduce**
1. Run the node in a VM.
2. Kill the VM (without shutting down the node first).
3. Restart the VM and the node within the VM.

**Expected behavior**
THe logs should continue to be displayed while the node is validating its on-disk storage.

**Screenshots and attachments**
The last log message before the seeming hang was:
```
[nix:cardano.node.ip-producers:Notice:5] [2021-08-28 04:29:22.40 UTC] IPSubscriptionTarget {ispIps = [], ispValency = 0}
```
then over 2 hour later the log was:
```
[nix:cardano.node.ChainDB:Info:5] [2021-08-28 04:35:11.25 UTC] Opened imm db with immutable tip at 5114891c5fe54befc11ae2e2d5f6ab74d086ecfce5f4a046cf5d25026f3f7982 at slot 38515068 and chunk 1783
```

**Comment by erikd** [comment]
Same issue. Log output of a standalone `node` (nothing else running in this VM):
```
[nix:cardano.node.dns-producers:Notice:5] [2021-09-11 07:36:57.94 UTC] [DnsSubscriptionTarget {dstDomain = "relays-new.cardano-mainnet.iohk.io", dstPort = 3001, dstValency = 1}]
[nix:cardano.node.ip-producers:Notice:5] [2021-09-11 07:36:57.94 UTC] IPSubscriptionTarget {ispIps = [], ispValency = 0}
```
the no output (or any signs of life) for over 30 minutes asnd then
```
[nix:cardano.node.ChainDB:Info:5] [2021-09-11 07:37:17.61 UTC] Opened imm db with immutable tip at 3b39fb8a5d97c18e79075bc8b1db12bdf64b7054f240d452d0e5ed4f012e63d0 at slot 39736285 and chunk 1839
[nix:cardano.node.ChainDB:Info:5] [2021-09-11 07:37:19.13 UTC] Opened vol db
```

**Comment by CarlosLopezDeLara** [comment]
Resolved since node 1.33.0 https://github.com/input-output-hk/cardano-node/releases/tag/1.33.0
